### PR TITLE
Fix sanity check prompt when new QPS benchmark scenarios are added

### PR DESCRIPTION
**Pull request by billfeng327**
Added direction on the scripts that need to be run when QPS benchmark scenarios are updated

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 1,965,108      Total (=)      1,965,108

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
10,887,891      Total (<)     10,887,894

 No significant differences in binary sizes


```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

File: tools/run_tests/sanity/check_qps_scenario_changes.py [hunk 1]
```diff
@@ -27,7 +27,9 @@
 json_run_localhost_bzl = 'test/cpp/qps/json_run_localhost_scenarios.bzl'
 
 if qps_json_driver_bzl in output or json_run_localhost_bzl in output:
-    print('qps benchmark scenarios have been updated, please commit '
+    print('qps benchmark scenarios have been updated, please go '
+          'to //test/cpp/qps directory, run json_run_localhost_scenario_gen.py '
+          'and qps_json_driver_scenario_gen.py scripts, and commit '
```
Comment (vjpai) [inline tools/run_tests/sanity/check_qps_scenario_changes.py]: Aren't these scripts run automatically from `tools/buildgen/generate_projects.sh` ? If not, wouldn't that be a better way to run them?

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 1,965,335      Total (=)      1,965,335

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
10,888,233      Total (>)     10,888,228

 No significant differences in binary sizes


```

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 1,965,335      Total (=)      1,965,335

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
10,888,227      Total (>)     10,888,224

 No significant differences in binary sizes


```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: tools/run_tests/sanity/check_qps_scenario_changes.py (hunk 1 above)
Comment (billfeng327) [reply tools/run_tests/sanity/check_qps_scenario_changes.py]: they are not run from generate_projects.sh
I was told that there is no pre-commit scripts for gRPC testing, so this is the best solution that I could think of

Comment (vjpai) [reply tools/run_tests/sanity/check_qps_scenario_changes.py]: Would it be worthwhile to add to tools/buildgen/generate_projects.sh ? That way it's just one place to fix things no matter what form of project change has happened and then the sanity checks will also notice that projects weren't generated properly.

**Pull request closed without merging**